src/contrib/inference/abstractmcmc.jl

```diff
@@ -3,28 +3,19 @@ struct TuringState{S,F}
     logdensity::F
 end
 
-struct TuringTransition{T,NT<:NamedTuple,F<:AbstractFloat}
-    θ::T
-    lp::F
-    stat::NT
-end
-
-function TuringTransition(vi::AbstractVarInfo, t)
+function Transition(vi::AbstractVarInfo, t)
```

Comment:
Two comments: This constructor should be defined next to https://github.com/TuringLang/Turing.jl/blob/e41f58cabf0022c53d7c6c4a0b1e73368c6b4e38/src/inference/Inference.jl#L135 (otherwise it will be confusing for anyone reading the code) - and it seems it is not tested, so could you add a test that covers it?

Reply:
Agreed! However, I am wrong to think this `Transition` would be tested when testing the external samplers?

Reply:
I assumed that as well, and I assumed we test some of them - but apparently these lines are not tested.